src/PointCloud/Core/PointCloudOctant.cs

```diff
@@ -1,4 +1,5 @@
-﻿using Fusee.Math.Core;
+﻿using Fusee.Engine.Core.Primitives;
```

Comment:
I don't think we should reference the `Primitives.Sphere` here (same goes for the comment in `InsideOrIntersectingSphere`) as we only need the mathematical construct "sphere" and not the renderable mesh.